Makefile

```diff
@@ -7,7 +7,7 @@ init:
 	git submodule update --init
 
 lint:
-	npx prettier@2.7.1 . --check
+	npx prettier . --check
```

Comment:
I prefer stick to a fixed version, as prettier may change style.

Reply:
right now it is v3, so let's bump to the latest?